pySDC/implementations/problem_classes/AllenCahn_1D_FD.py

```diff
@@ -229,16 +254,23 @@ def u_exact(self, t):
 
 
 class allencahn_front_semiimplicit(allencahn_front_fullyimplicit):
-    """
-    Example implementing the Allen-Cahn equation in 1D with finite differences and inhomogeneous Dirichlet-BC,
-    with driving force, 0-1 formulation (Bayreuth example), semi-implicit time-stepping
+    r"""
+    This class implements the one-dimensional Allen-Cahn equation with driving force using inhomogeneous Dirichlet
+    boundary conditions
 
-    Attributes
-    ----------
-    A : scipy.diags
-        Second-order FD discretization of the 1D laplace operator.
-    dx : float
-        Distance between two spatial nodes.
+    .. math::
+        \frac{\partial u}{\partial t} = \frac{\partial^2 u}{\partial x^2} - \frac{2}{\varepsilon^2} u (1 - u) (1 - 2u)
+            - 6 d_w u (1 - u)
+
+    for :math:`u \in [0, 1]`. Centered finite differences are used for discretization of the second order spatial derivative.
+    The exact solution is given by
+
+    .. math::
+        u(x, t) = 0.5 \left(1 + \tanh\left(\frac{x - vt}{\sqrt{2}\varepsilon}\right)\right)
+
+    with :math:`v = 3 \sqrt{2} \varepsilon d_w`. For time-stepping, this problem will be treated in a
+    *semi-implicit* way, i.e., the Laplacian is treated implicitly by solving the linear system including the
```

Comment:
maybe you can remove the "by solving the linear system including the Laplacian here